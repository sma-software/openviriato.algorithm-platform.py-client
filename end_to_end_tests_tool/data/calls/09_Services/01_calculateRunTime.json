{
    "restResource": "/services/trains/{trainId}:run-time-calculation",
    "httpMethod": "GET",
    "urlParameters": [ "=multiTrain:id=" ],
    "requestBody": null,
    "CSharpMethodName": "CalculateRunTimes(trainID)",
    "CSharpMethodParameterTypes": [ "System.Int64" ],
    "subcategory": "Calculate Running Times"
}
