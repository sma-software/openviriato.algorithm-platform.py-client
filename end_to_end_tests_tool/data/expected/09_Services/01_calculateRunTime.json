{"trainId":4974,"updateTimesTrainPathNodes":[{"trainPathNodeId":3880,"arrivalTime":"2003-05-01T07:00:00","departureTime":"2003-05-01T07:00:00","minimumRunTime":null,"minimumStopTime":"P0D","stopStatus":"commercialStop"},{"trainPathNodeId":4246,"arrivalTime":"2003-05-01T07:01:42","departureTime":"2003-05-01T07:01:42","minimumRunTime":"PT1M42S","minimumStopTime":"P0D","stopStatus":"passing"},{"trainPathNodeId":4610,"arrivalTime":"2003-05-01T07:04:38","departureTime":"2003-05-01T07:04:38","minimumRunTime":"PT2M56S","minimumStopTime":"P0D","stopStatus":"commercialStop"}]}